packages/grid/src/components/row/Row.types.ts

```diff
@@ -0,0 +1,20 @@
+import { ReactNode } from 'react';
+
+type BaseAlignmentOptions = 'start' | 'end' | 'center';
+
+type JustifyOptions =
+  | BaseAlignmentOptions
+  | 'space-between'
+  | 'space-evenly'
+  | 'space-around';
+
+type AlignOptions = BaseAlignmentOptions | 'stretch' | 'baseline';
+
+export type RowProps = {
+  children: ReactNode;
+  className?: string;
+  element?: keyof JSX.IntrinsicElements;
+  justifyContent?: JustifyOptions;
+  alignItems?: AlignOptions;
```

Comment:
I would suggest to rename to more shorter
`alignItems` to `align` 
and
`justifyContent ` to `justify`